var/spack/repos/builtin/packages/libsonata/package.py

```diff
@@ -17,6 +17,8 @@ class Libsonata(CMakePackage):
     git = "https://github.com/BlueBrain/libsonata.git"
 
     version('develop', branch='master', submodules=False, get_full_repo=True)
+    version('0.1.9.1', commit='59b298e', submodules=False, get_full_repo=True)
+    version('0.1.9', tag='v0.1.9', submodules=False, get_full_repo=True)
```

Comment:
Perhaps there shouldn't be a `v0.1.9` at all, or we should re-tag the version again with a different commit that points to `59b298e`.

Reply:
Yeah, agree that v0.1.9 shouldn't exist, unless it points to `59b298e`, which is a bit dirty; otherwise we need to do a release, which seems like overkill

Reply:
In CoreNEURON what we do is to release temporary versions without the tags, kind of `v0.1.9.20210721` (with the date appended at the end). For instance:

https://github.com/BlueBrain/spack/pull/1173/files#diff-292c6c2abbfcea3f168434d209f8978cfcdf06f56b7c07eb513b17c2d506e33bR23

This way, in Spack, we could omit the 0.1.9 release altogether with a comment and add a new one `v0.1.9.20210721`.

What do you think?

Reply:
I'm agnostic on the whole situation, but it seems like a reasonable compromise; perhaps w/ a comment saying why it's being done this way?

Reply:
Perfect! Let's wait for the CI to pass, I will upload a change with a proper comment explaining the situation, and we take it from there.